app/views/trips/edit.erb

```diff
@@ -0,0 +1,71 @@
+x<div class="containter">
+  <h1>Update Trip</h1>
+</div>
+
+<div class="container">
+
+<form action="/trips/<%= @trip.id %>" method="POST">
+
+  <div class="form-group">
+    <input type="hidden" name="_method" value="PUT"/>
+  </div>
+
+  <h4>Edit Starting information</h4>
+  <div class="form-group">
+    <label for="start_station">Start Station: (Currently <%= Station.find(@trip.station_id).name %>)</label>
+    <select class="form-control" id="start_station" name="trip[start_station_name]">
+      <% @stations.each do |station| %>
+        <option><%= station.name %></option>
+      <% end %>
+    </select>
+  </div>
+
+
+  <div class="form-group">
+    <label>Start Date: (Current Start Date <%= @trip.start_date %>)</label>
+    <input type="datetime-local" name="trip[start_date]" class="form control"><br>
+    <p><strong>Please enter same or new date!</strong></p><br>
+  </div>
+
+  <h4>Edit Ending information</h4>
+
+  <div class="form-group">
+    <label for="end_station">End Station: (Currently <%= Station.find(@trip.end_station_id).name %>)</label>
```

Comment:
same - we don't want to query our db in our views - that's the responsibility of our controllers